opendbc/car/nissan/interface.py

```diff
@@ -14,6 +14,7 @@ def _get_params(ret: structs.CarParams, candidate, fingerprint, car_fw, alpha_lo
     ret.brand = "nissan"
     ret.safetyConfigs = [get_safety_config(structs.CarParams.SafetyModel.nissan)]
     ret.autoResumeSng = False
+    ret.steerAtStandstill = True
```

Comment:
It's easiest to merge smaller, standalone changes separately. Right now you're gating this feature addition on a safety related lateral torque change, which obviously takes much longer to review + requires comma to buy or rent the vehicle to validate it ourselves.

Can you split this change out and get a route of steering at a standstill?

Reply:
@sshane https://connect.comma.ai/5255045de19f57a2/00000041--5b19d68d6c